frontend/src/scenes/cohorts/cohortEditLogic.ts

```diff
@@ -176,6 +183,8 @@ export const cohortEditLogic = kea<cohortEditLogicType>([
                         return processCohort(cohort)
                     } catch (error: any) {
                         lemonToast.error(error.detail || 'Failed to fetch cohort')
+
+                        actions.setCohortMissing()
```

Comment:
@mariusandra would be a great extension to kea-loaders if they automatically set a `xxxFailed` value like `loadCohortFailed` ...